app/Cardano/Shell/Features/Logging.hs

```diff
@@ -1,5 +1,6 @@
-{-# LANGUAGE DeriveGeneric #-}
-{-# LANGUAGE Rank2Types    #-}
+{-# LANGUAGE DeriveGeneric     #-}
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
This is included in the cabal as the default extension. Or it should be included.